src/NuGet.Core/NuGet.ProjectModel/TargetFrameworkInformation.cs

```diff
@@ -22,6 +22,11 @@ public class TargetFrameworkInformation : IEquatable<TargetFrameworkInformation>
         public IList<NuGetFramework> Imports { get; set; } = new List<NuGetFramework>();
 
         /// <summary>
+        /// If True AssetTargetFallback behavior will be used for Imports.
+        /// </summary>
+        public bool AssetTargetFallback { get; set; }
```

Comment:
This is written to the project.json equivalent, the rest of the properties aren't in this style currently. I'm going to leave this as the msbuild property equivalent for now.